src/tests/cmocka/test_kcm_marshalling.c

```diff
@@ -167,8 +167,8 @@ static void test_kcm_ccache_marshall_unmarshall_binary(void **state)
     uint8_t *data;
     uuid_t uuid;
 
-    owner.ucred.uid = getuid();
-    owner.ucred.gid = getuid();
+    cli_creds_set_uid(&owner, getuid());
+    cli_creds_set_gid(&owner, getgid());
```

Comment:
Note: this was `getuid()` and I replaced it with `getgid()`. Not sure if this was a typo or intentional.

Reply:
Thanks. I think it was a typo.